src/store/state/state-model.ts

```diff
@@ -49,10 +50,10 @@ export const stateModel: StateModelIF = {
     filingId: null
   },
   nameTranslations: [],
-  incorporationDateTime: {
+  effectiveDateTime: {
     valid: false,
-    isFutureEffective: false,
-    effectiveDate: null
+    isFutureEffective: null,
+    dateTimeString: ''
```

Comment:
The difference here is that you can do `''.slice()` but you can't do `null.slice()`.